contracts/src/rollup/RollupCore.sol

```diff
@@ -190,6 +190,15 @@ abstract contract RollupCore is IRollupCore, PausableUpgradeable {
         return _stakerMap[staker].amountStaked;
     }
 
+    /**
+     * @notice Get the withdrawal address of the given staker
+     * @param staker Staker address to lookup
+     * @return Withdrawal address of the staker
+     */
+    function withdrawalAddress(address staker) public view override returns (address) {
+        return _stakerMap[staker].withdrawalAddress;
```

Comment:
should check if the staker exists

Reply:
the other ones like `amountStaked` don't so i'm not sure this one should